api/failure_detector.cc

```diff
@@ -81,15 +80,10 @@ void set_failure_detector(http_context& ctx, routes& r, gms::gossiper& g) {
     });
 
     fd::get_endpoint_phi_values.set(r, [](std::unique_ptr<request> req) {
-        std::map<gms::inet_address, gms::arrival_window> map;
+        // FIXME: this was based on gms::get_arrival_samples.
+        // But the latter returned an empty std::map<inet_address, arrival_window>.
+        // The functionality needs to be implemented by the failure detector.
```

Comment:
We no longer have a phi failure detector, so the functionality will never be implemented.
Maybe we should just remove the API. Even if someone depends on it (I doubt it) it wasn't returning anything meaningful for some time already.

Reply:
We do not need the FIXME, just returning the empty value is good enough.  The phi values associated with the old fd is gone and is not going back.